doc/source/notebooks/advanced/fast_predictions.pct.py

```diff
@@ -64,6 +64,8 @@
 # \begin{equation*}
 #     \alpha = [K_{mm} + \sigma^2I]^{-1}\mathbf{y}\\ Q^{-1} = [K_{mm} + \sigma^2I]^{-1}
 # \end{equation*}
+# _(note in practice, we cache the cholesky decomposition of Q)_
```

Comment:
It would be great to have a brief explanation of why this is the case.

Reply:
Thanks...good suggestion...done